sagemaker_processing/fairness_and_explainability/fairness_and_explainability.ipynb

```diff
@@ -2,658 +2,21 @@
  "cells": [
```

Comment:
<p>Instead of redirecting in the notebooks (this will make the number of notebooks unmanageable), can you search through this Github for links to it and change it to the new one? Also work with your writer to search for any URLs in the AWS Docs and replace it with the new one as well. You should do this for all the old notebooks.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3720/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
we followed this approach as there will be customer who might have bookmarked older links. so redirecting them would be ideal. 
I will follow up with our writer regarding updating the AWS Docs with the new links.